README.md

```diff
@@ -1 +1,17 @@
-# ttt
+# Tic Tac Toe in Clojure
+
+This project uses: 
+
+ - PostgreSQL 11.5 running on localhost, port 5432
```

Comment:
Great!  This is really helpful, because now I know what needs to be installed and running.
One more question on this: Do I need to create a database and a user before running TTT, or does the TTT program do that automatically?